kogito-kafka-quickstart-quarkus/README.md

```diff
@@ -113,14 +190,31 @@ there are bunch of extension attributes that starts with `kogito` to provide som
 
 To take the other path of the process put following message on `travellers` topic
 
+* Send Message that should be skipped to Topic
+```
+kafka_2.11-2.4.0/bin/kafka-console-producer.sh --broker-list localhost:9092 --topic travellers
 ```
-{ 
-"firstName" : "John", 
-"lastName" : "Doe", 
-"email" : "john.doe@example.com", 
-"nationality" : "American"
+
+With the following content (Cloud Event Format)
+```
+{
+  "specversion": "0.3",
+  "id": "31627e26-31eb-43e7-8343-92a696fd96b1",
+  "source": "",
+  "type": "VisaApplicationsMessageDataEvent_8", 
+  "time": "2019-10-01T12:02:23.812262+02:00[Europe/Warsaw]",
+  "data": { 
+	"firstName" : "John", 
+	"lastName" : "Doe", 
+    "email" : "john.doe@example.com", 
+    "nationality" : "American"
+	}
 }
+```
 
+One Liner
+```
+{"specversion": "0.3","id": "31627e26-31eb-43e7-8343-92a696fd96b1","source": "","type": "VisaApplicationsMessageDataEvent_8", "time": "2019-10-01T12:02:23.812262+02:00[Europe/Warsaw]","data": { "firstName" : "John", "lastName" : "Doe", "email" : "john.doe@example.com", "nationality" : "American"}}
```

Comment:
same here, should be `TravellersMessageDataEvent_3`

Reply:
fixed @radtriste